src/main/kotlin/com/bookmarks/models/User.kt

```diff
@@ -0,0 +1,14 @@
+package com.bookmarks.models
+
+data class User(
+    val id: UInt,
+    val name: String,
```

Comment:
What is the difference between name, surname and nickname? :) I can understand the difference between surname and nickname, but what would be the `name`?

Reply:
Oops, it should have been `firstname`.